www/_posts/2017-10-09-tools-release.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+author:
+    name: Steve Gill
+    url: https://twitter.com/stevesgill
+title:  "Tools Released!"
+categories: news
+tags: release tools
+---
+
+We just released a small update to our tools!
+
+* [cordova@7.1.0](https://www.npmjs.org/package/cordova)
+* [cordova-lib@7.1.0](https://www.npmjs.org/package/cordova-lib)
+* [cordova-plugman@1.5.1](https://www.npmjs.org/package/plugman)
+* [cordova-fetch@1.2.0](https://www.npmjs.org/package/cordova-fetch)
+* [cordova-common@2.1.1](https://www.npmjs.org/package/cordova-common)
+* [cordova-node-xcode@1.0.0](https://www.npmjs.org/package/xcode)
+* [cordova-js@4.2.2](https://www.npmjs.org/package/cordova-js)
+
+## Release Highlights
+
+* [CB-13303](https://issues.apache.org/jira/browse/CB-13303) added `--production` flag by default. This means that when the commands `cordova platform add android` or `cordova plugin add cordova-plugin-device` are run, under the hood we are running `npm install android --production`. The `--production` flag only installs `dependencies` from `package.json` and skips `devDependencies`.  This should speed up installs (especially when installing local copies of platforms and plugins). You can turn the flag off by passing the `--noprod` flag or setting it off globally via `cordova config set production false`.
```

Comment:
should say `npm install cordova-android --production` instead of `npm install android --production`